<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRecvUdpClass</class>
 <widget class="QWidget" name="QRecvUdpClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRecvUdp</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="pMainLayout" stretch="0,0,1">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1,0,0">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Interface : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="pIfBox"/>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Host :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pHostEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Port :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pPortEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="pShortCheck">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>short log</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="pWriteCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>write to :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pPathEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="pSelectPathBtn">
            <property name="toolTip">
             <string>select write path</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pProcessBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="pOutList">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pIfBox</tabstop>
  <tabstop>pHostEdit</tabstop>
  <tabstop>pPortEdit</tabstop>
  <tabstop>pShortCheck</tabstop>
  <tabstop>pWriteCheck</tabstop>
  <tabstop>pPathEdit</tabstop>
  <tabstop>pSelectPathBtn</tabstop>
  <tabstop>pProcessBtn</tabstop>
  <tabstop>pOutList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
